### ISSUE TITLE
Default keyboard shortcut for Pan map 

### ISSUE THREAD

**Issue by frspp** [qgis/QGIS#41280]
Hi!

There is no default keyboard shortcut for Pan map. I suggest 'p' like pan or 'h' like hand. H is free by default. 

qgis 3.16.3
macos 11.1

**Comment by rduivenvoorde** [comment]
@frspp I agree with you that it seems that 'keyboard shortcuts' are never assigned by looking at, for example, usability....
Both p and h are already used (p for print layout and h for 'show QGIS home page)... while a very important task like panning does NOT have a shortcut

I think all is personal preference (and in QGIS case more often 'historic reasons', and you should for now use the standard functionality of QGIS to assing you preferred shortcut to pan if you want. (Note that using a 3 button mouse, the easiest to navigate is to not use the buttons at all, but just use your mouse to pan and middlemouse button to zoom in).

note: these shortcuts are easily overridden with translations... Make p not obvious for [random language] speakers etc etc

A thorough discussion about usability features like this seems (and IS) very difficult  :-(
